src/ui/aliases_main_area.ui

```diff
@@ -163,10 +163,10 @@
       </size>
      </property>
      <property name="toolTip">
-      <string>&lt;p&gt;Type in one or more commands you want the alias to send directly to the game if the keys entered match the pattern. (Optional)&lt;/p&gt;&lt;p&gt;To send more complex commands, that could depend on or need to modifies variables within this profile a Lua script should be entered &lt;i&gt;instead&lt;/i&gt; in the editor area below.  Anything entered here is, literally, just sent to the game server.&lt;/p&gt;&lt;p&gt;It is permissible to use both this &lt;i&gt;and&lt;/i&gt; a Lua script - this will be sent &lt;b&gt;before&lt;/b&gt; the script is run.&lt;/p&gt;</string>
+      <string>&lt;p&gt;Enter one or more commands to send directly to the game if the keys match the pattern. (Optional)&lt;/p&gt;&lt;p&gt;Alternatively, you can enter a replacement alias that will be executed instead of the original one. For complex commands that require modification of variables within this profile, use a Lua script in the editor area below &lt;i&gt;instead&lt;/i&gt;. Anything entered here is sent directly to the game server. It's possible to use both this field &lt;i&gt;and&lt;/i&gt; a Lua script - the contents of this field will be sent &lt;b&gt;before&lt;/b&gt; running the script.&lt;/p&gt;</string>
```

Comment:
```suggestion
      <string>&lt;p&gt;Enter one or more commands to use if the given command matches the pattern. (Optional)&lt;/p&gt;&lt;p&gt;This could be another alias or a command to send directly to the game. For complex commands that require modification of variables within this profile, use a Lua script in the editor area below &lt;i&gt;instead&lt;/i&gt;. It's possible to use both this field &lt;i&gt;and&lt;/i&gt; a Lua script - the contents of this field will be used &lt;b&gt;before&lt;/b&gt; running the script.&lt;/p&gt;</string>
```